admin/docker/Dockerfile.icpc

```diff
@@ -28,15 +28,20 @@ RUN ln -s /usr/local/kotlinc/bin/* bin/
 
 # Reconfigure problemtools:
 # - Use PyPy for Python 2
+# - Use PyPy for Python 3
 # - Use /usr/local/bin rather than /usr/bin for Kotlin
 RUN mkdir -p /etc/kattis/problemtools
 RUN echo " \n\
 python2: \n\
```

Comment:
Python2 is no longer supported at the worldfinals:
https://icpc.global/worldfinals/programming-environment

Reply:
True, but it is possible that judges may still want to use it as a solution language.